### DISCUSSION TITLE
Welcome to Refugees-Support Discussions!

### DISCUSSION THREAD

**Discussion by mattmartin26** [Cath-Strategic-Tech/Refugees-Support#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NOMANZONE** [comment]
My Asylum experience in USA

I was born in a notorious dictatorship country with shocking human rights problems. I am highly educated (educated in usa), not yet win Nobel Prize, but most people in the world didn't have my achievements. I had been persecuted for 8 years before resorted to asylum.

 I claimed asylum according to the law at the port of entry at Niagara Fall .

The CBP there quickly called the embassy of my country of origin and had me wait for long hours till night and told me I had only 2 options, 1 is to go back to where I came from, 2 is to go to jail because NY does not have detention center. Of course, I can't go back, so the police handcuffed me after a thorough body search done to me. In darkness and in a black car (it looks like regular car from outside, but same as police car inside), once I was in jail, all my belongings were taken away, including my underwear. 

At Chautauqua jail, The guards ordered me to take off all my clothes in front of them to be completely naked. The guards ordered me to squat and split my ass for them to see so they could whisper and comment to each other for fun.

I was imprisoned among thieves, drug dealers and strippers, who sometimes purposely made problems for me but it was always me, the victim, to be punished. I never made problems to others, but I was repeatedly locked 24 hours alone in my cell. The water tasted like bleach, most of the inmates had constipation because of the food. This jail specially put me into the coldest room of the entire jail. This jail purposely did not provide warm clothes or blankets, the sheet with holes they gave me had a piss smell. I was in a frozen status everyday. The guards forced me to take cold water shower everyday. I can't make any phone calls because my contact books and money were taken away. Even if I had money and my contacts numbers, I still could not make phone calls, because the jail manipulated my inmate code, so I can only watch others having money to buy things and making phone calls chatting with friends, while I was locked in isolation. All my requests were purposely ignored. The only thing I was allowed to do was to write letters which I think for sure will be inspected by this jail before sent out, and I suspect my letters were never sent out.

One day, I got a phone call from the State department, and I mentioned the abuses at this jail, later I was transferred to federal prison which was better, but soon again, I was transferred back to Chautauqua and went through the ass-checking again. I saw the immigration officer often came to talk to the mexicans about their judge hearing, but they never talked to me. I was simply thrown into jail as a criminal and locked here ignored. I could not make phone calls to anyone such as lawyers. I never had any hearing and nobody knew that I was in jail.

I was tortured physically and psychologically for about 3 months. One day the ICE said they would release me but actually directly took me out of the jail in a car and deported me.
